src/Microsoft.Health.Dicom.Functions/Update/Models/UpdateInstanceBlobArguments.cs

```diff
@@ -16,14 +17,25 @@ public sealed class UpdateInstanceBlobArguments
 {
     public int PartitionKey { get; }
 
+    public Partition Partition { get; set; }
+
     public IReadOnlyList<InstanceFileState> InstanceWatermarks { get; }
```

Comment:
I'd mark any properties that we won't use in the future with `[Obsolete]` so that we remember